charts/camunda-platform/Chart.yaml

```diff
@@ -1,41 +1,59 @@
 apiVersion: v2
 name: camunda-platform
-version: 8.0.11
+version: 8.0.12
 appVersion: "8.0.0"
-description: Camunda Platform Helm Chart for Kubernetes
+description: |
+  Camunda Platform 8 Self-Managed Helm charts.
+  Camunda's process automation platform allows developers to design, automate and improve processes.
 type: application
-icon: https://helm.camunda.io/imgs/self-managed.png
+icon: https://helm.camunda.io/imgs/camunda.svg
```

Comment:
:question: Hm why we changed that

Reply:
I believe that the self-managed icon doesn't fit here because it doesn't provide a unique identifier for the Helm chart.
At the end of the day, the Helm chart is for Camunda Platform, which is the main product of Camunda.

For example, if you take a look at the chart in Artifact Hub (and probably it will show the same in any artifacts repo) ... it's hard to identify the package visually as a Camunda product.

![camunda-platform](https://user-images.githubusercontent.com/6760103/182806563-c17332bf-3bc3-4142-ae6b-6ec6f35b0220.png)

For these reasons, I believe that we should use the Camunda official logo.

Reply:
Ok :)